src/i18n/en.json

```diff
@@ -115,6 +115,10 @@
     "transaction.ColonyClient.moveFundsBetweenPotsWithProofs.title": "Move Funds Between Pots",
     "transaction.ColonyClient.moveFundsBetweenPotsWithProofs.description": "Move Token from one domain pot to another",
 
+    "transaction.group.setUserRoles.title": "Set user roles",
+    "transaction.group.setUserRoles.description": "Set user roles",
+    "transaction.ColonyClient.setUserRolesWithProofs.title": "Set user roles",
+    "transaction.ColonyClient.setUserRolesWithProofs.description": "Set user roles",
     "transaction.group.setRoles.title": "Set roles",
     "transaction.group.setRoles.description": "Sign the transactions required to set the selected roles.",
```

Comment:
If you've added new message descriptors for the new roles actions, please make sure to remove the old ones